.travis.yml

```diff
@@ -1,6 +1,12 @@
 sudo: false
 language: python
-python: 3.5
+matrix:
+  include:
+    - python: 3.5
+    - python: 3.6
+    - python: 3.7
+      dist: xenial
+      sudo: true
```

Comment:
is sudo really required ?

Reply:
Yes, that's what I read on the Internet.

Reply:
Yes i confirm, sudo is required on xenial